pages/docker_compose.md

```diff
@@ -91,25 +91,57 @@ If you just want to start your database, and not the other services, use the Doc
 
 ### Cassandra
 
-Before running `docker-compose -f src/main/docker/app.yml up`, you will need to build and set up manually your Cassandra image.
+#### In development
+To start a Cassandra cluster to run your application locally, you can use the docker_compose file for development `cassandra.yml`:
+`docker-compose -f src/main/docker/cassandra.yml up -d`
 
-**Using DataStax OpsCenter**: if you want to monitor your application using DataStax OpsCenter, we also provide a `cassandra-opscenter.yml` configuration file, which you should use instead of the normal `cassandra.yml` file. OpsCenter will be available at [http://localhost:8888](http://localhost:8888).
+Docker-compose will start 3 services:
+- **tatami-cassandra**, a container with the Cassandra node contact point
+- **tatami-cassandra-node-1**, a container with a second Cassandra node joining the cluster
+- **tatami-cassandra-migration** a container to automatically apply all migrations scripts (create the Keyspace, create the tables, all data migrations, ...)
 
-To use Cassandra, we need to build and run a specific Docker image:
+The migration service is responsible to apply all the migration scripts from src/main/resources/config/cql in the following order:
 
-- Build the image: `docker-compose -f src/main/docker/cassandra.yml build`
-- Start the container (it will show the container id): `docker-compose -f src/main/docker/cassandra.yml up`
-- Use `docker ps` to get the "container id" of the jhipster-cassandra node, which will be used in the next commands (replace `<container_id>` by its value)
-- Copy the cql scripts inside the container: `docker cp src/main/resources/config/cql/ <container_id>:/`
-- Initialize the database by creating the Keyspace and the Tables: `docker exec -it <container_id> init`
-- After using the [entity sub-generator]({{ site.url }}/creating-an-entity/), update the cql scripts inside the container: `docker cp src/main/resources/config/cql/ <container_id>:/`
-- Create the tables: `docker exec -it <container_id> entities`
+1. `create-keyspace.cql`
+2. `create-tables.cql` - the initial bootstrap script.
+3. all `cql/migration/\*.cql` files in alphabetical order - scripts altering the schemas or data (including `*_added_entity_*.cql` files generated by `jhipster:entity`). This is where you will add all the database changes
```

Comment:
it could be intesting to add this command.

After generating new entities, we can launch this:
`docker-compose -f src/main/docker/cassandra.yml up <app>-cassandra-migration`